manifest.php

```diff
@@ -39,7 +39,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '32.11.2',
+    'version'     => '32.11.3',
```

Comment:
Since there is a dependency to another PR, could you also declare the required version here?